### DISCUSSION TITLE
Table - EditAction->Modal onother Model?

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#9612] [Help]
How it is possible, to make an Action to edit an another Model Type,  inside a ResourceTable.

```
                Action::make('edit_another_model')
                    ->label('Edit')
                    ->record(fn ($record) => AnotherModel::find(444))
                    ->form(static::anotherModelForm())
```

When i click on it, it tries to open the Form, but with that Model Type from the main Ressource.

**Comment by tiagof** [comment]
Would love an answer of this as well.
